cypress/e2e/felony/utils/testIneligiblePage.ts

```diff
@@ -0,0 +1,19 @@
+export default function testIneligiblePage(link: string) {
+  describe("f-classa-main-1-ineligble", () => {
+    it('has ineligibility content', () => {
+      cy.visit(`/calculator/${link}`);
+      cy.contains('Your conviction is not eligible to vacate')
+        .should('be.visible')
```

Comment:
As we dont want to be too much coupled on Sanity texts, i would suggest we to remove this, because we dont want, for example, that our test break if anyone on Sanity change `is not` for `isn't`.

We just wat to make sure real necessary content are in this page
```suggestion
```